metadata-db-app/src/migrations/054_fix_catalog_acl_concept_ids.clj

```diff
@@ -0,0 +1,35 @@
+(ns migrations.054-fix-catalog-acl-concept-ids
+  (:require [clojure.java.jdbc :as j]
+            [clojure.edn :as edn]
+            [cmr.common.util :as util]
+            [config.migrate-config :as config]
+            [config.mdb-migrate-helper :as h]
+            [cmr.metadata-db.data.oracle.providers :as providers]
+            [clojure.string :as string]))
+
+(defn up
+  "Migrates the database up to version 54."
+  []
+  (println "migrations.054-fix-catalog-acl-concept-ids up...")
+  (doseq [result (h/query "select * from cmr_acls where acl_identity like 'catalog-item%'")]
+    (println result)
+    (let [{:keys [id metadata deleted]} result
+          deleted (= 1 (long deleted))
+          ;;Remove bad parts of metadata string
+          metadata (-> metadata
+                       (util/gzip-blob->string)
+                       (string/replace #"Executing.*\n" "")
```

Comment:
Would really like to see some unit tests that take corrupted sample data and show this cleaning it up ...